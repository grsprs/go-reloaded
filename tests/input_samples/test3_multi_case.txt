transform these three words (cap, 3) properly