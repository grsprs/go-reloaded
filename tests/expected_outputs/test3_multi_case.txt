Transform These Three words properly